server/graphql/types/src/types/requisition.rs

```diff
@@ -232,6 +232,21 @@ impl RequisitionNode {
         Ok(RequisitionLineConnector::from_vec(result))
     }
 
+    pub async fn program_name(&self) -> &Option<String> {
+        &self.requisition.program_name
+    }
+
+    pub async fn order_type(&self) -> &Option<String> {
+        &self.row().order_type
+    }
+
+    pub async fn period(&self) -> Option<PeriodNode> {
+        match &self.requisition.period {
```

Comment:
nit: self.requisition.period.clone().map(PeriodNode::from_domain), this is purely stylistic, just giving another option.